PowerEditor/src/functionList.xml

```diff
@@ -1294,6 +1295,22 @@
 						"
 				/>
 			</parser>
+			
+			<!-- ============================================== [ BaanC Langage ] -->
+			<parser
+				displayName="BaanC"
+				id         ="baanc_function"
+				commentExpr="(?m-s:/|.*?$)"
+			>
+				<function
+						mainExpr="^[\t\x20]*(function)\s+[\w\.\s]+\(" displayMode="$functionName"

```

Comment:
Replace `[\t\x20]*` with `\h*`.
Replace `(function)` with `function` or `(?i:function)` to force case-insensitive match or `(?-i:function)` to force case-sensitive match.
Remove the `displayMode` attribute as it isn't used.